pages.es/common/gradle.md

```diff
@@ -11,14 +11,14 @@
 
 `gradle build -x {{test}}`
 
-- Ejecuta en modo offline para prevenir que gradle acceda a la red durante una compilación:
+- Ejecuta en modo offline para prevenir que Gradle acceda a la red durante una compilación:
 
 `gradle build --offline`
 
 - Limpia el directorio de compilación:
 
 `gradle clean`
 
-- Compila y genera un paquete:
+- Genera una APK en modo lanzamiento:
```

Comment:
I have to define the English version (below) before changing the Spanish version... :sob: